content/en/_index.html

```diff
@@ -296,9 +296,11 @@ <h5>November 21, 2022</h5>
       <div class="tag">Blog</div>
     </div>
     <p class="font-size-sm">
-      Notary and Notation Notary project is a set of tools that helps you sign,
+      Notation is a CLI and set of libraries that helps you sign,
```

Comment:
Notary Project is a set of open source specifications that have a reference implementation in Notation CLI and its associated libraries.

Reply:
I think we should change to: 
> Notary Project is a set of open-source specifications and tools for securing software supply chains. Notation CLI and associated libraries are an implementation of the Notary Project signing specification.

Reply:
fixed